src/appComponents/ErrorFallback.tsx

```diff
@@ -0,0 +1,193 @@
+import Alert from '@mui/material/Alert';
+import Container from '@mui/material/Container';
+
+import { FC, useState } from 'react';
+
+import { UserFeedback } from './types';
+
+// DEFAULTS
+const TITLE = 'Sorry, something went wrong with this application';
+const FORM_TITLE =
+  'Our team has been notified. If you would like to help, please, tell us what happened below.';
+const ERROR_DETAILS = 'Details of the error';
+const NAME_LABEL = 'Name';
+const NAME_HELPER = 'Provide your name (optional)';
+const EMAIL_LABEL = 'Email';
+const EMAIL_HELPER = 'Provide your email (optional)';
+const COMMENT_LABEL = 'Comment';
+const COMMENT_HELPER = 'Tell us what happened (optional)';
+const THANKS_FOR_FEEDBACK = 'Thank you for your feedback!';
+const SEND = 'Send your feedback';
+
+interface ErrorFallbackProps {
+  error: Error;
+  componentStack: string;
+  eventId: string;
+  captureUserFeedback: (userFeedback: UserFeedback) => void;
+  title?: string;
+  formTitle?: string;
+  nameLabel?: string;
+  nameHelper?: string;
+  emailLabel?: string;
+  emailHelper?: string;
+  commentLabel?: string;
+  commentHelper?: string;
+  thanksMessage?: string;
+  sendButtonLabel?: string;
+  errorDetailsLabel?: string;
+}
+
+/**
+ * 
+ * @returns A form to submit user feedback.
+ * 
+ * @example
+const ErrorBoundary: FC<{ children?: ReactNode }> = ({ children }) => {
+  const { t: tFallback } = useTranslation('translations', {
+    keyPrefix: 'ERROR_BOUNDARY.FALLBACK',
+  });
+  return (
+    <Sentry.ErrorBoundary
+      // eslint-disable-next-line react/no-unstable-nested-components
+      fallback={({ error, componentStack, eventId }) => (
+        <ErrorFallback
+          error={error}
+          componentStack={componentStack}
+          eventId={eventId}
+          captureUserFeedback={Sentry.captureUserFeedback}
+          title={tFallback('MESSAGE_TITLE')}
+          formTitle={tFallback('MESSAGE_FEEDBACK')}
+          nameLabel={tFallback('NAME_LABEL')}
+          nameHelper={tFallback('NAME_HELPER')}
+          emailLabel={tFallback('EMAIL_LABEL')}
+          emailHelper={tFallback('EMAIL_HELPER')}
+          commentLabel={tFallback('COMMENT_LABEL')}
+          commentHelper={tFallback('COMMENT_HELPER')}
+          thanksMessage={tFallback('THANKS_FOR_FEEDBACK')}
+          sendButtonLabel={tFallback('SEND')}
+          errorDetailsLabel={tFallback('ERROR_DETAILS')}
+        />
+      )}
+    >
+      {children}
+    </Sentry.ErrorBoundary>
+  );
+};
+ */
+const ErrorFallback: FC<ErrorFallbackProps> = ({
+  error,
+  componentStack,
+  eventId,
+  captureUserFeedback,
+  title = TITLE,
+  formTitle = FORM_TITLE,
+  nameLabel = NAME_LABEL,
+  nameHelper = NAME_HELPER,
+  emailLabel = EMAIL_LABEL,
+  emailHelper = EMAIL_HELPER,
+  commentLabel = COMMENT_LABEL,
+  commentHelper = COMMENT_HELPER,
+  thanksMessage = THANKS_FOR_FEEDBACK,
+  sendButtonLabel = SEND,
+  errorDetailsLabel = ERROR_DETAILS,
+}) => {
```

Comment:
We prefer the style that does not use the `FC`, I know we switched a while back and you used to declare them like that, and you were right :)
```suggestion
const ErrorFallback = ({
  error,
  componentStack,
  eventId,
  captureUserFeedback,
  title = TITLE,
  formTitle = FORM_TITLE,
  nameLabel = NAME_LABEL,
  nameHelper = NAME_HELPER,
  emailLabel = EMAIL_LABEL,
  emailHelper = EMAIL_HELPER,
  commentLabel = COMMENT_LABEL,
  commentHelper = COMMENT_HELPER,
  thanksMessage = THANKS_FOR_FEEDBACK,
  sendButtonLabel = SEND,
  errorDetailsLabel = ERROR_DETAILS,
}: ErrorFallbackProps): JSX.Element => {
```